tox.ini

```diff
@@ -0,0 +1,32 @@
+[tox]
+skipsdist=True
+envlist=pycodestyle,pylint,yamllint
+
+[testenv]
+deps =
+  -cconstraints.txt
+  -raws/requirements.txt
+
+[testenv:pylint]
```

Comment:
It's typically nicer to consolidate various linting tools under pre-commit, like this: https://github.com/ansible/pylibssh/blob/bfc1e97/.pre-commit-config.yaml + https://github.com/ansible/pylibssh/blob/bfc1e97/tox.ini#L296-L309.

Reply:
Thanks for the suggestion. It's a good idea but slightly out of scope for this PR.